recipe/meta.yaml

```
@@ -8,22 +8,22 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/platformdirs-{{ version }}.tar.gz
-  sha256: 58c8abb07dcb441e6ee4b11d8df0ac856038f944ab98b7be6b27b2a3c7feef19
-
+  sha256: b45696dab2d7cc691a3226759c0d3b00c47c8b6e293d96f6436f733303f77f6d
 build:
   number: 0
-  # Trigger 1
-  skip: True  # [py<37]
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
-    - hatchling >=0.22.0
-    - hatch-vcs
+    - wheel
+    - hatchling >=0.17.1
```

```suggestion
    - hatchling >=1.17.1
```
https://github.com/platformdirs/platformdirs/blob/3.10.0/pyproject.toml#L5